# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

HandBrakeActionGoal action_goal
HandBrakeActionResult action_result
HandBrakeActionFeedback action_feedback
